etna/core/models/mixins.py

```diff
@@ -133,4 +138,38 @@ class Meta:
         )
     ]
 
+
+class RequiredHeroImageMixin(models.Model):
+    """Mixin to add hero_image attribute to a Page, and make it required."""
+
+    hero_image = models.ForeignKey(
+        get_image_model_string(),
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    hero_image_caption = RichTextField(
+        verbose_name="hero image caption (optional)",
+        features=["bold", "italic", "link"],
+        blank=True,
+        help_text=(
+            "An optional caption for hero images. This could be used for image sources or for other useful metadata."
+        ),
+    )

```

Comment:
```suggestion
class RequiredHeroImageMixin(HeroImageMixin):
    """Mixin to add hero_image attribute to a Page, and make it required."""
```

Reply:
Hi @ababic, same comment as this one: https://github.com/nationalarchives/ds-wagtail/pull/970#discussion_r1221444578